capstone/src/components/User/useSettings.jsx

```diff
@@ -74,17 +60,12 @@ export default function useSettings(userType, username) {
         setActiveIDs(res.data.active);
         setInactiveIDs(res.data.inactive);
       });
-      // ONLY FOR ADVENTURER
       if (isAdventurer) {
-        getPriorization(username, userType).then((res) =>
-          setPrioritize(res.data),
-        );
-        // ONLY FOR ADVENTURER
-        getActiveMinValues(username).then((res) =>
-          setMinPreferenceValues(res.data),
-        );
-        // ONLY FOR ADVENTURER
-        getBoolMinValues(username).then((res) => setHasMinValues(res.data));
+        getPreferenceRestrictions(username).then((res) => {
```

Comment:
Just keep in mind that every time this hook is called, we'll send out the request. The state isn't shared *between* hooks.

Reply:
Thanks for that!!